metaflow/plugins/conda/conda_step_decorator.py

```diff
@@ -213,13 +214,36 @@ def runtime_init(self, flow, graph, package, run_id):
         self.metaflow_home = tempfile.mkdtemp(dir='/tmp')
         os.symlink(path_to_metaflow, os.path.join(self.metaflow_home, 'metaflow'))
 
+        # Do the same for metaflow_custom
+        try:
+            import metaflow_custom as m
+        except ImportError:
+            # No additional check needed because if we are here, we already checked
+            # for other issues when loading at the toplevel
+            pass
+        else:
+            custom_paths = list(m.__path__)
+            if len(custom_paths) == 1:
+                # Regular package
+                os.symlink(custom_paths[0], os.path.join(self.metaflow_home, 'metaflow_custom'))
+            else:
+                # Namespace package; we don't symlink but add the additional paths
+                # for the conda interpreter
+                self.addl_paths = [os.path.split(p)[0] for p in custom_paths]
```

Comment:
L:217 - L:232 shouldn't be related to the escape hatch. I believe rebasing on master should get rid of this diff.